### DISCUSSION TITLE
Use remarkable to replace marked

### DISCUSSION THREAD

**Discussion by mzy2240** [writer/writer-framework#10] [Ideas]
Remarkable is much faster than marked (3-8X), with even smaller gzipped size.

**Comment by ramedina86** [comment]
I've looked at remarkable but doesn't seem well-maintained, their last commit was almost two years ago and adoption has plummeted. I'm also a bit skeptical that markdown performance would make a difference in the overall user experience (I'm open to being proven wrong though).

**Comment by mzy2240** [comment]
I did not realize it becomes inactive for that long. Then it is reasonable to not use it.
